test/functional/interface_http.py

```diff
@@ -48,13 +48,13 @@ def run_test(self):
         conn.request('POST', '/', '{"method": "getbestblockhash"}', headers)
         out1 = conn.getresponse().read()
         assert b'"error":null' in out1
-        assert conn.sock!=None  #according to http/1.1 connection must still be open!
+        assert conn.sock is not None  #according to http/1.1 connection must still be open!
 
         #send 2nd request without closing connection
         conn.request('POST', '/', '{"method": "getchaintips"}', headers)
         out1 = conn.getresponse().read()
-        assert b'"error":null' in out1  #must also response with a correct json-rpc message
-        assert conn.sock!=None  #according to http/1.1 connection must still be open!
+        assert b'"error":null' in out1 #must also response with a correct json-rpc message
```

Comment:
14903: pls revert changes on this line

Reply:
Thank you @UdjinM6 implemented review comment